### DISCUSSION TITLE
Fuel tank shading acting weird. Anyone else having this issue?

### DISCUSSION THREAD

**Discussion by Jdance-Media** [SmartlyDressedGames/Unturned-3.x-Community#3939] [Q&A]
![2023-06-03_09-40-22](https://github.com/SmartlyDressedGames/Unturned-3.x-Community/assets/67397422/d9dda0dc-1f00-4113-be11-81470586c56d)
I know that on many of the models, many faces are not connected for whatever reason. I'd rather not make an issue post for this, since it might just be me and my graphics settings.

__Graphics Settings:__
![Untitled452788](https://github.com/SmartlyDressedGames/Unturned-3.x-Community/assets/67397422/3a060966-3ecf-4f17-a704-676708a0fe2b)

**Comment by Wyvernaed** [comment]
I think it looks like all or almost all meshes in unturned have unity autosmooth with legacy normals mode.
![image](https://github.com/SmartlyDressedGames/Unturned-3.x-Community/assets/113861259/bcb39866-9c04-4d72-98e2-38dcaeba6983)

Solution for this would be switch normals mode to "Unweighted".
